rasa/shared/nlu/training_data/features.py

```diff
@@ -289,11 +289,11 @@ def combine(
                 tuple(f.origin) if not isinstance(f.origin, Text) else (f.origin,)
                 for f in features_list
             )
-            if len(origins) > 1:
-                raise ValueError(
-                    f"Expected all Features to have the same origin "
-                    f"found the following origins: {origins}."
-                )
+            # if len(origins) > 1:
```

Comment:
Why is this commented?

Reply:
Bug in the finetuning validator. Kathrin fixed it [here](https://github.com/RasaHQ/rasa/pull/9689) (will merge `main` later into the integration branch)